music/src/index.ts

```diff
@@ -18,6 +18,8 @@
 import * as tf from '@tensorflow/tfjs-core';
 export {tf};
 
+export * from './core/constants';
```

Comment:
I don't think you should be adding these here. Instead, you should use mm.constants.CONSTANT_NAME, etc.

Reply:
i had originally tried that but it didn't work, giving me the error:
`TypeScript error: callback_player.ts(99,23): Error TS2339: Property 'constants' does not exist on type 'typeof "../src/index"'.`

given that the import line is `import * as mm from '../src/index';` it makes sense that it can't find it unless i add that line.

Reply:
gotcha, and how do you find it after you add the line? is it mm.constants.*?

Reply:
no need for `constants`, just `mm.HI_CLICK_PITCH` for instance.

Reply:
we want these to be under mm.constants actually, same with mm.data

Reply:
how do you do this with typescript? i tried `export * from './core/constants' as constants;` (and a few variants) but it didn't work.
or should i just import these explicitly in callback_player?

Reply:
Ah, I think you need to add them to ./core/index.ts. In fact, the data export is already there.

Reply:
cool, thanks! done.